notebooks/official/stable_diffusion/dreambooth_stablediffusion.ipynb

```diff
@@ -0,0 +1,1326 @@
+{
```

Comment:
<p>Consider this comment from Discord:</p><p> </p><p> </p><p>Even though the original paper recommends using 4 to 6 images, the community in Discord has found that using&nbsp;<strong>10 to 12 images</strong>&nbsp;leads to better results.Oct 26, 2022</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
That's a good point and matches my experience. That said, we don't have a ready-to-use dataset that we can provide to customers. Do you have any suggestions?